test.c

```diff
@@ -2,14 +2,12 @@
 
```

Comment:
Is test.c used other than for a scratch area for trying stuff out? Should it be in the repo?